src/datanode/src/region_server.rs

```diff
@@ -1104,6 +1112,7 @@ enum RegionChange {
     Register(RegionAttribute),
     Deregisters,
     Catchup,
+    Update,
```

Comment:
The `Update` here can be confusing in future. How about changing to `Data`, which means the change is a *Data Change*?